tests/pkg_micro-ecc/Makefile

```diff
@@ -0,0 +1,13 @@
+APPLICATION = micro-ecc
+include ../Makefile.tests_common
+
+# boards without hardware random number generator
+BOARD_BLACKLIST := chronos msb-430 msb-430h qemu-i386 telosb wsn430-v1_3b wsn430-v1_4 z1 samr21-xpro
```

Comment:
The line `FEATURES_REQUIRED = periph_random` should suffice. (And a quick `make buildtest` without this line agrees.)